logos/logo-gen.sh

```diff
@@ -0,0 +1,6 @@
+sizes=('16' '32' '48' '64' '128' '256' '512' '1024')
+
+for size in ${sizes[*]}
```

Comment:
not the correct file but since the svg logo moved to the `logos` directory we should update the readme

Reply:
Oh right !